lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -3650,13 +3650,9 @@ function prepareQuestionDialog() {
 	    $('#question-error').text(msg("simplepage.question-need-2"));
 	    $('#question-error-container').show();
 	    return false;
-	} else if ($("#shortanswerSelect").prop("checked") && $("#question-graded").prop("checked") &&
-		   $(".question-shortanswer-answer").filter(function(index){return $(this).val()!=="";}).length < 1) {
-	    $('#question-error').text(msg("simplepage.question-need-1"));
```

Comment:
Do you think we could also remove the translation keys for that message?

Reply:
We can but not sure people want to remove the restriction.

Reply:
Ok, but if we remove it then we should get rid of the key.

Reply:
Okay!